codex_of_the_damned/templates/archetypes/bibliodeque/akunanse-classic.json

```diff
@@ -0,0 +1,225 @@
+{
+  "name": "Akunanse Classic",
```

Comment:
Could add "author": "Bibliodèque", it should get displayed in the deckles between the title and the deckles.

Reply:
Currently using `player` to have it closer to deck name